### ISSUE TITLE
Running `nx reset` with WebStorm open fails

### ISSUE THREAD

**Issue by NoahAndrews** [nrwl/nx#26796]
### Current Behavior

If I run `nx reset` with the workspace open in WebStorm, I get a "Failed to clean up the native file cache" error

### Expected Behavior

It should successfully clear the cache without having to close WebStorm

### GitHub Repo

https://github.com/nrwl/nx-examples

### Steps to Reproduce

1. Open the project in WebStorm
2. Run `yarn install`
3. Run `nx build my-app`
4. Refresh the workspace in the Nx Console WebStorm pane
5. Run `nx reset`

### Nx Report

```shell
Node   : 20.14.0
OS     : win32-x64
yarn   : 1.22.22

nx                 : 19.4.0-rc.0
@nx/js             : 19.4.0-rc.0
@nx/jest           : 19.4.0-rc.0
@nx/linter         : 19.4.0-rc.0
@nx/eslint         : 19.4.0-rc.0
@nx/workspace      : 19.4.0-rc.0
@nx/angular        : 19.4.0-rc.0
@nx/cypress        : 19.4.0-rc.0
@nx/devkit         : 19.4.0-rc.0
@nx/eslint-plugin  : 19.4.0-rc.0
@nx/react          : 19.4.0-rc.0
@nrwl/tao          : 19.4.0-rc.0
@nx/web            : 19.4.0-rc.0
@nx/webpack        : 19.4.0-rc.0
typescript         : 5.4.3
---------------------------------------
Registered Plugins:
@nx/eslint/plugin
@nx/cypress/plugin
@nx/jest/plugin
---------------------------------------
Community plugins:
@ngrx/component-store : 18.0.1
@ngrx/effects         : 18.0.1
@ngrx/entity          : 18.0.1
@ngrx/operators       : 18.0.1
@ngrx/router-store    : 18.0.1
@ngrx/store           : 18.0.1
@ngrx/store-devtools  : 18.0.1
```

### Failure Logs

```shell
NX   Resetting the Nx cache and stopping the daemon.



 NX   Failed to reset the Nx workspace.

Failed to clean up the native file cache.
```

### Operating System

- [ ] macOS
- [ ] Linux
- [X] Windows
- [ ] Other (Please specify)

**Comment by MaxKless** [comment]
Hey! Sorry about this, it's a revert from previous behaviour.  

Here's a PR that will fix it https://github.com/nrwl/nx/pull/26806

**Comment by MaxKless** [comment]
Hey! This should be fixed with `19.4.1`, please let me know if you run into any other issues in the future :)
